homework-1/src/index.js

```diff
@@ -0,0 +1,251 @@
+      <li>George Washington</li>
+      <li>John Adams</li>
+      <li>Thomas Jefferson</li>
+    </ul>
+    <ol start="4">
+      <li>James Madison</li>
+      <li>James Monroe</li>
+      <li>John Quincy Adams</li>
+    </ol>
+    <ul>
+      {arrayOfPresidentsFrom7To9.map((president) =>
+        <li key={president}>
+          {president}
+        </li>
+      )}
+    </ul>
+    <ul style={stylingForListOfPresidentsFrom10To14}>
+      {listOfPresidentsFrom10To14.map(president =>
+        president.presidentIndex % 2 === 1 ? (
+          <li key={president.presidentIndex}>
+            {formatedListOfPresidentsFrom10To14(president)}
+          </li>
+        ) : null
+      )}
+    </ul>
+    <ul>
+      {dataFromServer
+        .sort((a, b) => (a.date > b.date ? 1 : -1))
+        .map(event => {
+          return (
+            <li key={event.id}>
+              <a
+                href={`https://www.facebook.com/events/${event.id}/`}
+                target='_blank'
+                rel="noopener noreferrer">
+                {event.title}
+              </a>
+              <p>{getDay(event.date) + ', ' + getTime(event.date)}</p>
+              <p>{event.place}</p>
+            </li>
+          );
+        }
+        )}
+    </ul>
+
+    <form className='form' onSubmit={handleSubmit}>
+      <fieldset>
+        <div className='form__item'>
+          <label htmlFor='name'>Name</label>
+          <input
+            id='name'
+            type='text'
+            name='name'
+            pattern='[A-Za-z]{1,32}'
+            title='Your name should only contain upper and lowercase letters (e.g. Ivan)'
+            placeholder='Put your name'
+            minLength='2'
+            required
+            onChange={handleChange} />
+        </div>
+        <div className='form__item'>
+          <label htmlFor='password'>Password</label>
+          <input
+            id='password'
+            type='password'
+            name='password'
+            pattern='.{6,}'
+            title='Your name should only contain 6 or more characters'
+            placeholder='Put your password'
+            minLength='4'
+            required
+            onChange={handleChange} />
+        </div>
+        <div className='form__item'>
+          <label>Basic plan</label>
+          <input
+            type='radio'
+            name='plan'
+            value='basic'
+            defaultChecked
+            onChange={handleChange} />
+        </div>
+        <div className='form__item'>
+          <label>Premium plan</label>
+          <input
+            type='radio'
+            name='plan'
+            value='premium'
+            onChange={handleChange} />
+        </div>
+        <div htmlFor='newsletter' className='form__item'>
+          <label>Subscribe to our newsletters</label>
+          <input
+            id='newsletter'
+            type='checkbox'
+            name='newsletter'
+            defaultChecked
+            onChange={handleChange} />
```

Comment:
For checkbox you'll always get one value, please fix it

Reply:
Added improvements according to code review